crdt_lib/include/list.h

```diff
@@ -40,18 +40,27 @@ namespace nupad::crdt {
 
         void applyInsertOperation(Operation &operation) {
             auto *insertOp = dynamic_cast<InsertOperation<T> *>(&operation);
-            CHECK_NE(insertOp, nullptr) << "Cannot cast operation to InsertOperation: " << operation;
```

Comment:
Change this to CHECK_NE(op, nullptr) to avoid compilation errors

Reply:
CHECK_NOTNULL works